### ISSUE TITLE
Shields still showing blocked items even when Allowed

### ISSUE THREAD

**Issue by ryanbr** [brave/brave-browser#13129]
## Description 
Allowing ads in shields, still showing items being blocked on https://amp.cnn.com/cnn/2020/12/09/politics/republican-reaction-trump-refusal-to-concede/index.html

## Steps to Reproduce

   1.  Using Nightly, open `https://amp.cnn.com/cnn/2020/12/09/politics/republican-reaction-trump-refusal-to-concede/index.html`
   2.  Open shileds, Allow ads/tracking
   3.  Refresh, and still show 4 blocked items.

## Actual result:

![ads-trackers](https://user-images.githubusercontent.com/1659004/101716489-3a5bcc00-3b02-11eb-8d63-6a66b8213476.png)

## Expected result:
Allow trackers and ads, if user wants. Show no blocked items.

## Reproduces how often: 

Easily in recent Night Build

## Brave version (brave://version info)
`Version 1.20.20 Chromium: 88.0.4315.5 (Official Build) nightly (64-bit)`

## Version/Channel Information:

- Can you reproduce this issue with the current release?  No
- Can you reproduce this issue with the beta channel? No
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

```
Nightly Version 1.20.17  (Good)
Nightly Version 1.20.20  (Bad)
```

**Comment by mikhoul** [comment]
Same here but with the stable release.

**Comment by rebron** [comment]
@ryanbr I'm not seeing this anymore in 1.23.71. Can you recheck? I believe we addressed this via another issue.

<img width="1068" alt="Screen Shot 2021-04-16 at 10 05 11 AM" src="https://user-images.githubusercontent.com/5951041/115059320-3fa79980-9e9b-11eb-951b-d2926c315ec5.png">

**Comment by ryanbr** [comment]
Closing, no longer applicable.
